src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -552,9 +552,7 @@ public void testAllCheckstyleMessages() throws Exception {
                     .isEqualTo(Modifier.PUBLIC | Modifier.STATIC | Modifier.FINAL);
 
                 // below is required for package/private classes
-                if (!message.isAccessible()) {
-                    message.setAccessible(true);
-                }
+                message.trySetAccessible();
```

Comment:
Ignoring inputs, does this only impact Fields or do we need to do methods and constructors?
https://github.com/checkstyle/checkstyle/search?q=setAccessible

Reply:
My fault, resolved.

Reply:
We really only need to replace usages where we checked if something was accessible first to avoid exception.